### ISSUE TITLE
Limit number of announcements sent for a given study per day

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#576]
**TL;DR**: Don't send more than X announcements for a given study in a day, so we don't "use them up" the first day the study launches.

**Narrative**
"As a Researcher, I may start my study, then realize there is some minor problem that I want to fix. During the time I'm fixing it, participants see my study is not available, and may not try again; because we don't want to bug them again about the same study, I will have "used up" my notifications in that initial launch.

"As a Researcher, if many many participants are notified about my study when it first launches, I may quickly reach the target sample size too quickly."

"As a Participant, I may see an email about a new study and think that would be fun to try this weekend. But when I go to participate in it, it's no longer available. I would rather hear less often but only about studies that are actually available." 

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [x] There is a maximum number of emails about any given study that will be sent when announcement emails are sent out. (Reasonable value is probably around 50-100 at this point.)
- [x] Test for not sending >X emails about given study

**Implementation Notes**

* Setting a hard-coded cap X of emails/study would be fine for now.
* As a crude approximation (because I think this would be easier?), we could just cap the number of total emails sent (as X * number of active public studies). If we're selecting randomly from possible emails this would work a bit differently for studies with more vs. fewer eligible participants. E.g. if we'll only send 50 emails/study/day, study A has 40 eligible participants total, and study B has 3000 eligible participants, then with a simple cap per study we'd email all 40 for study A the first day. With a combined cap we'd email on average 1.2 participants per day about study A (because we'd choose 100 out of 3040 emails, 40 of which are about study A). I think this would be harder to reason about at least.
* Could also make this cap a field on the study with a reasonable default, even if we don't provide a way for researchers to edit it at this point; I could imagine at some point it being useful for researchers to be able to adjust the pace of contacting eligible participants.
* By "per day" I really mean "per email-sending task" (i.e. if we also switch to only sending emails once a week we just want some cap)
